apps/sms/views/conversation/js/conversation.js

```diff
@@ -580,13 +580,12 @@ var ConversationView = {
       // Populate draft if there is one
       // TODO merge with handleDraft ? Bug 1010216
```

Comment:
nit: Maybe you'll need to update the bug number once you create another bug for handleDraft